web/src/components/molecules/Request/Details/Comment.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "@emotion/styled";
+import moment from "moment";
```

Comment:
Consider at some point moving over to dayjs. I believe moment isn't maintained any more, so I am using dayjs in what I'm working on now and it works well!
You can resolve this comment after you read it